stl/inc/ostream

```diff
@@ -187,6 +205,76 @@ ios_base::iostate _Print_noformat_unicode(ostream& _Ostr, const string_view _Str
     return _State;
 }
 
+template <class _Filebuf_type = filebuf>
+ios_base::iostate _Print_newline_only_unicode(ostream& _Ostr) {

```

Comment:
This is slightly modified from `_Print_noformat_unicode` above, and contains massive complicated machinery. We should extract the commonality.